lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3681,6 +3698,14 @@ namespace {
       if (forceClassMethod && decl->hasRelatedResultType())
         return nullptr;
 
+      // Hack: avoid importing methods named "print" that aren't available in
+      // the current version of Swift. We'd rather just let the user use
+      // Swift.print in that case.
+      if (!isActiveSwiftVersion() &&
+          isPrintLikeMethod(importedName.getDeclName(), dc)) {
```

Comment:
Good question. It has to be something that users type instinctively that a method ends up shadowing. `min` and `max` probably also fall into that category, but those seem like much rarer Objective-C method names. I'd rather keep it limited for now.